external/radiation/python/radiation_driver.py

```diff
@@ -118,6 +118,7 @@ def radinit(
                     f"- ISUBCLW={isubclw}, Use McICA with random ",
                     f"permutation seeds for LW random number generator",
                 )
+                raise ValueError('isubclw == 2 is not implemented')
```

Comment:
This should raise an error if `isubclw` is _not_ 2. Same issue with `isubcsw`. You could keep the print statements for options 0 and 1 and just add `raise` statements after them.